src/email_relay/migrations/0002_auto_20231030_1304.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 4.2.5 on 2023-10-30 13:04
+
+from django.db import migrations
+
+from email_relay.conf import EMAIL_RELAY_DATABASE_ALIAS
+
+
+def migrate_message_data_to_new_schema(apps, schema_editor):
+    """Migrate the JSON data from the old schema to the new schema.
+
+    Changes:
+        - "message" is now "body"
+        - "recipient_list" is now "to"
+        - "html_message" is now a part of "alternatives", a list of tuples of (content, mimetype)
+
+    Adds:
+        - "cc"
+        - "bcc"
+        - "reply_to"
+        - "extra_headers"
```

Comment:
Small nitpick: Are there `headers` too? The other fields made sense to me because they man more 1:1 with the function signatures or what comes out of the emails. Does `extra_headers` too?

Reply:
https://docs.djangoproject.com/en/4.2/topics/email/#django.core.mail.EmailMessage

> headers: A dictionary of extra headers to put on the message. The keys are the header name, values are the header values. It’s up to the caller to ensure header names and values are in the correct format for an email message. **The corresponding attribute is extra_headers.**

Reply:
🤣  I saw that, but then the example made me question it:

```python
email = EmailMessage(
    "Hello",
    "Body goes here",
    "from@example.com",
    ...
    headers={"Message-ID": "foo"},
)
```

Reply:
More a question than anything that should hold It up.

Reply:
Ah, I see that now. Yeah I dunno, I could go either way. I went with `extra_headers` because that's what you access on an `EmailMessage` 🤷‍♂️

Reply:
https://github.com/django/django/blob/9b18af4f6f12b9d25157e0b5afc3dca198f6dd06/django/core/mail/message.py#L250C38-L250C38

Reply:
Let's keep it then. If it keeps you up at night, we can always change it, and we probably will need to keep extra_headers for API support either way.